modules/ROOT/pages/anypoint-filter-processor.adoc

```diff
@@ -77,7 +77,7 @@ The Anypoint Filter processor allows you to filter messages according to the fol
 |*Filter expired* a|
 This option allows you to define an expiration time for the message.
 
-* To define an expiration time based upon the time a message entered the flow, enter a Mule Expression to call a variable that provides this date and time in http://en.wikipedia.org/wiki/ISO_8601[ISO 8601] format (yyyy-MM-dd'T'hh:mm:ssZ), and an expiration time in milliseconds. (Earlier in the flow, you will need to name and define this variable so that it extracts the date and time information from your message's metadata.) The expiration time will be calculated based on the timestamp extracted from the message, and the filter will discard messages received after that expiration time.
+* To define an expiration time based upon the time a message entered the flow, enter a Mule Expression to call a variable that provides this date and time in ISO 8601 format (yyyy-MM-dd'T'hh:mm:ssZ), and an expiration time in milliseconds. (Earlier in the flow, you will need to name and define this variable so that it extracts the date and time information from your message's metadata.) The expiration time will be calculated based on the timestamp extracted from the message, and the filter will discard messages received after that expiration time.
 * To define an exact date and time after which messages should expire, enter a date and time in http://en.wikipedia.org/wiki/ISO_8601[ISO 8601] format (yyyy-MM-dd'T'hh:mm:ssZ), and an expiration time in milliseconds. The expiration time is calculated from the precise date and time you enter, and the filter will discard messages received after that expiration time.
```

Comment:
There is one missing 
> http://en.wikipedia.org/wiki/ISO_8601[ISO 8601]
here.

Reply:
@fermujica There are many. Unfortunately, I don't have 4 hours to fix all of them. I just fixed as many as I could in 1/2 an hour. The main thing is fixing the broken link.